packages/manager/apps/dedicated/client/app/license/license.controller.js

```diff
@@ -141,6 +142,10 @@ export default /* @ngInject */ (
 
   $scope.resetAction();
 
+  $scope.letterCase = (content) => {
+    return capitalize(content);
+  };
```

Comment:
May be you can avoid using external dependency in this case.
You could use :
1 - Bootstrap helper text-capitalize (see how it works)
2 - Using CSS `text-transform: capitalize;`

Reply:
Interesting but I want to capitalize only the "fallback translation", I don't want to change the format of the translation when it exists